allure-generator/src/main/javascript/translations/isv.json

```diff
@@ -151,7 +151,12 @@
       "name": "Minule proby"
     },
     "severity": {
-      "name": "Važnost"
+      "name": "Važnost",
+      "blocker": "kritična",
```

Comment:
Would it be acceptable to translate "Blocker" as "Critical" and "Critical" as "Major" in some translations? The phrase "**Blocking** **_importance_**" does not sound to my ear right.

Reply:
I think, better to use `blocker` here as well because `kritična` more applicable for `critical`